test/operator.jl

```diff
@@ -162,4 +162,81 @@ using Compat.Test
         @test_throws ErrorException q * q2
         @test_throws ErrorException q / q2
     end
+
+    @testset "Higher-level operators" begin
+        @testset "sum" begin
+            sum_m = Model()
+            @variable(sum_m, 0 ≤ matrix[1:3,1:3] ≤ 1, start = 1)
+            # sum(j::JuMPArray{Variable})
```

Comment:
These comments aren't necessary. We don't name the method being tested in the rest of the tests. If we want to keep them, we can have named testsets instead.